authentication/views_test.py

```diff
@@ -33,84 +44,322 @@ def test_auth_views_disabled(settings, client, user, url):
     assert response.status_code == status.HTTP_404_NOT_FOUND
 
 
-@pytest.mark.parametrize('user_password,input_password,expected_state,is_active,is_authenticated,expected', (
-    # not logged in, password matches, should succeed
-    ('password1', 'password1', SocialAuthState.STATE_SUCCESS, True, False, True),
-    # logged in, password matches, should succeed
-    ('password1', 'password1', SocialAuthState.STATE_SUCCESS, True, True, True),
-    # not logged in, password matches, user inactive, should fail
-    ('password1', 'password1', SocialAuthState.STATE_INACTIVE, False, False, False),
-    # not logged in, password doesn't match, user active, should fail
-    ('password1', 'password2', SocialAuthState.STATE_ERROR, True, False, False),
-))
-def test_login_email_and_password(
-        settings, client, user, user_password, input_password, expected_state, is_active, is_authenticated, expected
-):  # pylint: disable=too-many-arguments
-    """Tests email/password login flow"""
-    settings.FEATURES[features.EMAIL_AUTH] = True
-    user.set_password(user_password)
-    user.is_active = is_active
-    user.save()
-
-    if is_authenticated:
-        client.force_login(user)
+def last_partial_token():
+    """Returns the most recent partial token"""
+    return Partial.objects.last().token
```

Comment:
Do we need an `order_by`? The `Partial` table doesn't seem to have a default ordering

Reply:
this is unused now, so i'll just remove the whole func